src/hammer-vlsi/test.py

```diff
@@ -616,6 +619,80 @@ def test_bumps(self) -> None:
          shutil.rmtree(tech_dir_base)
          shutil.rmtree(test.run_dir)
 
+    def test_pins(self) -> None:
```

Comment:
I appreciate this comprehensive test.

Reply:
Yeah it took an annoying amount of time because of the stackup not being in a common helper method, causing the import change, breaking mysteriously because of weird python stuff. But it is done and now no one else can break it. :)